src/views/components/ItemLink.js

```diff
@@ -14,11 +14,11 @@ const ItemLink = ({
     instanceId,
     holdingsRecordId,
     itemId,
-    item: { barcode },
+    item,
```

Comment:
What did we remove it for?

Reply:
When request is `Title` level, we have no item in it and it become `undefined`. There is an error when we will try to take `item.barcode`. But now `item.barcode` will be executed only when we guaranteed got `item`.